packages/react/src/components/Button/index.tsx

```diff
@@ -26,6 +26,8 @@ const Button = forwardRef<HTMLButtonElement, ButtonProps>(
         'Button--secondary': variant === 'secondary',
         'Button--error': variant === 'error',
         Link: variant === 'link',
+        Tag: variant === 'tag',
+        'Button--tag': variant === 'tag',
```

Comment:
Rather than introduce a new class name, can we just leverage `button.Tag` like we are doing for `button.Link`?

Reply:
I could go either way on this, although we should be consistent on both. I think we can reutilize the styles for `.Tag` but it would be nice to specifically target our buttons with `.Button--variant` classname. If we want to use `.Button--tag`, I think `.Button--link` should exist as well.

Reply:
@scurker My personal preference would be `.Button.Tag`, since it still targets our buttons without introducing another classname that needs to be included any time you want to do `className="Button Tag"`. However, like you said, I wanted to be consistent with what we had w/o introducing a breaking change (`button.Link` -> `.Button.Link`) 🤷

Reply:
Thoughts?

Reply:
@thuey The problem is css specificity, and is counter to [our guidelines](https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md#styles): 

> For modifiers, such as variants or states, these values would be delimited by two dashes --

```css
.Calendar--variant-large {
  font-size: var(--text-size-large);
}

.Calendar--variant-small {
  font-size: var(--text-size-small);
}
```

Reply:
@scurker 
I see what you are saying. I think part of the weirdness we are running into here is that `Tag` itself is an "element", but it's also being used as a variant.

In any case, it seems like our options are:
1. Leave it `button.Tag` and `button.Link` to have both consistency and not introduce a breaking change.
2. Change to `.Button--tag` and `.Button--link` to have both consistency, but introduce a breaking change.
3. Change to `.Button--tag` but keep `button.Link` to get the tag variant to be how we want it, not introduce a breaking change, but lose consistency.

I'm leaning toward option 1, but would be good with whichever you think is best.

Reply:
I'm suggesting something completely different that doesn't introduce a breaking change:

* Use both `button.Tag` and `button.Button--tag`. `.Tag` inherits the styles from `.Tag` and `.Button--tag` keeps our conventions in place (and implementing the same pattern for `.Link`).

This allows someone to target either all tags (with `.Tag`) or just button tag variants (with `.Button--tag`) while keeping specificity simple.

Reply:
Gotcha. That works for me 👍

Reply:
Actually, that still doesn't answer the question of what happens to the current `button.Link` styles:

https://github.com/dequelabs/cauldron/blob/62c917271fd30df2e4e65c8eb421578ae20f373c/packages/styles/button.css#L44-L46

https://github.com/dequelabs/cauldron/blob/62c917271fd30df2e4e65c8eb421578ae20f373c/packages/styles/button.css#L231-L237

If we change those to `button.Button--link`, that is a breaking change. If we leave them as-is, that breaks consistency.

Reply:
Since this is an unrelated change to the proposed work, I don't have an issue with this being a breaking change as part of a separate ticket so here's my suggestion:

* Create a separate issue to change the `.Link` class to `.Button--link`
* Make the change here for `.Button--tag` as a `feat`

Reply:
Here's the ticket: https://github.com/dequelabs/cauldron/issues/681

Sorry for the confusion @AdnoC. We will want to do `Button--tag` after all